x11-misc/j4-dmenu-desktop/j4-dmenu-desktop-2.17.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+MY_PV="r${PV}"
+MY_P="${PN}-${MY_PV}"
+
+DESCRIPTION="A fast desktop replacement for i3-dmenu-desktop"
+HOMEPAGE="https://github.com/enkore/j4-dmenu-desktop"
+SRC_URI="https://github.com/enkore/j4-dmenu-desktop/archive/${MY_PV}.tar.gz -> ${MY_P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-3+"
+SLOT="0"
+IUSE="test"
+
+DEPEND="test? ( dev-cpp/catch:0 )"
+RDEPEND="x11-misc/dmenu"
+
+S="${WORKDIR}/${MY_P}"
+
+src_prepare() {
+	cmake-utils_src_prepare
+
+	# Respect users CFLAGS
+	sed -i -e "s/-pedantic -O2//" CMakeLists.txt
```

Comment:
missing `|| die`

Reply:
*sigh* .. I need a repoman for this x( Thanks!

Reply:
It was there before.

Reply:
Yes ;-)